http/handlers.go

```diff
@@ -65,7 +65,7 @@ func writeJSON(jsonableResponse interface{}, w http.ResponseWriter, statusCode i
 	var json, err = json.MarshalIndent(jsonableResponse, "", " ")
 
 	if err == nil {
-		w.Write(json)
+		_, _ = w.Write(json)
```

Comment:
?

Reply:
Reverted to keep sonar code smell and process it correctly when we'll have time